andela/ah-frontend-targaryen
# #161776758 View all articles

#### What does this PR do?

Allow all users to view articles
Create a dashboard for our application
#### Description of Task to be completed?

Currently, our application has no UI for viewing articles and a dashboard

This task sass allows us to create a dashboard and UI to view all articles


#### How should this be manually tested?

Run the application `npm start`
On the default page/ landing page of the application all articles are visible
Login and you should be able to see both the dashboard and all articles

#### What are the relevant pivotal tracker stories?

[#161776758](https://www.pivotaltracker.com/story/show/161776758)



#### Any background context you want to add?
N/A



#### Screenshots
![image](https://user-images.githubusercontent.com/9901011/48246850-0b923980-e402-11e8-954f-f8893a1e2333.png)
![image](https://user-images.githubusercontent.com/9901011/48246912-5318c580-e402-11e8-9b3e-5b8b593a8d1b.png)

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`


  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
